### ISSUE TITLE
JSON DDL request failed with the following error: Cannot resolve all the paths while de-serializing Database

### ISSUE THREAD

**Issue by savoy9** [microsoft/Analysis-Services#56]
Trying to deploy a fairly substantial update to our production model, after a successful traditional publish and full process in Dev. ALM-TK gives a passing validation 
![image](https://user-images.githubusercontent.com/41710547/82299044-dc9b4300-9969-11ea-8dd9-9e40097c2b4b.png)
but I get this error message immediately on trying to deploy:

```
The JSON DDL request failed with the following error: Cannot resolve all the paths while de-serializing Database. .

Technical Details:
RootActivityId: 50d43571-89bf-4e0c-adad-5ba92734aed6
Date (UTC): 5/19/2020 7:39:47 AM
```

**Comment by christianwade** [comment]
As discussed offline, we have a repro with TMSL script and forwarded to engineering ...

**Comment by christianwade** [comment]
As discussed, fixed in 5.0.6
